src/plugins/usage_collection/public/index.ts

```diff
@@ -21,7 +21,7 @@ import { PluginInitializerContext } from '../../../core/public';
 import { UsageCollectionPlugin } from './plugin';
 
 export { METRIC_TYPE } from '@kbn/analytics';
-export { UsageCollectionSetup } from './plugin';
```

Comment:
This is here because I missed exporting it when I updated the UsageCollection plugin start contract here: #69836